src/zenml/steps/base_step.py

```diff
@@ -592,7 +592,7 @@ def __call__(
         )
 
         self.INPUT_SPEC = {
-            arg_name: artifact_type.type  # type:ignore[misc]
```

Comment:
hmm i hope mypy doesnt complain

Reply:
This somehow was in develop when I first branched off it. `mypy` complained that there was an unused `type:ignore` comment and told me to remove it. No idea how that got merged to `develop` in the first place.